docs/source/streamflow-public-testnet.md

```diff
@@ -211,8 +211,116 @@ The following [script](https://github.com/livepeer/pm-params-calculator) can use
+Receive 1 winning ticket every 36.92688330871492 hours
+```
+
+In the above example, if an orchestrator transcodes 10 streams into 240p, 360p and 720p renditions, sets the ticket EV to 1000 gwei, redeems winning tickets on-chain using a gas price of 5 gwei and sets the price per pixel to 1000 wei then the orchestrator will receive approximately .001354 ETH of value per hour and 1 winning ticket every ~36.929 hours.
+
 The script can also be used to calculate the price per pixel needed to target a desired frequency of winning tickets (in terms of hours) which can be useful for testing that your orchestrator can properly redeem winning tickets.
 
+```
+➜  pm-params-calculator git:(master) python3 calc.py -t
+DEFAULTS
+---------
+Ticket redemption gas cost: 100000
+Transaction cost overhead: 0.01
+
+
+Enter the desired ticket expected value (gwei): 1000
+Ticket expected value: 1000.0 gwei
+
+
+Enter the gas price (gwei) to use for ticket redemption transactions: 5
+Ticket redemption gas price: 5.0 gwei
+
+
+Transaction cost to redeem a ticket: 0.0005000000 ether ($0.1000000000)
+Ticket face value: 0.0500000000 ether ($10.0000000000)
+Ticket winning probability: 0.000020000000000
+1 out of 49999 tickets will win
+
+
+Enter the desired number of hours (i.e. 1, .5, etc.) until receiving a winning ticket: 1
+
+
+Would you like to add a rendition to be encoded? (y/n): y
+Enter the output width: 426
+Enter the output height: 240
+Enter the output FPS (frames per second): 30
+Ether the number of streams of this renditions: 10
+Would you like to add a rendition to be encoded? (y/n): y
+Enter the output width: 640
+Enter the output height: 360
+Enter the output FPS (frames per second): 30
+Ether the number of streams of this renditions: 10
+Would you like to add a rendition to be encoded? (y/n): y
+Enter the output width: 1280
+Enter the output height: 720
+Enter the output FPS (frames per second): 30
+Ether the number of streams of this renditions: 10
+Would you like to add a rendition to be encoded? (y/n): n
+
+
+Given the specified renditions you will and a target of 1 winning ticket every 1.0 hours you will:
+Need to charge 36911.09386590315 wei per pixel
+Encode 1354579200000.0 pixels per hour
+Receive 49999.0 tickets per hour
+Receive 0.0499990000 ether ($9.9998000000) (in terms of ticket expected value) per hour
+```
+
+In the above example, if an orchestrator transcodes 10 streams into 240p, 360p and 720p renditions, sets the ticket EV to 1000 gwei, redeems winning tickets on-chain using a gas price of 5 gwei and targets 1 winning ticket every hour then the orchestrator should set the price per pixel to 36911.09386590315 wei.
+
+In practice, the price set by orchestrators will likely also be influenced by the prices set by other orchestrators on the network as well as the orchestrator's own infrastructure cost. This information can be surfaced by tools that will be built in the future.
```

Comment:
Perhaps we could add that price currently doesn't impact selection, as long as all orchestrator's pricing in the pool is below the `maxPrice` for a broadcaster all O's in the pool have the same change of getting selected for the job if all other parameters are equal (equal stake for initial selection, below real-time latency). 

Additionally I would change "the price set by orchestrators will likely also be influenced by the prices set by other orchestrators on the network" to something like "the price set by orchestrators will likely also be influenced by supply and demand, benchmarks for pricing and market rates will become more accurate as network traffic grows".

Reply:
> Perhaps we could add that price currently doesn't impact selection, as long as all orchestrator's pricing in the pool is below the maxPrice

Added a note in 5eff245 about the current use of prices for filtering orchestrators.

> Additionally I would change "the price set by orchestrators will likely also be influenced by the prices set by other orchestrators on the network" to something like "the price set by orchestrators will likely also be influenced by supply and demand, benchmarks for pricing and market rates will become more accurate as network traffic grows".

Updated in 5eff245 to:

"In practice, the price set by orchestrators will likely also be influenced by the market rate for transcoding on the network and an orchestrator's own infrastructure cost. This information can be surfaced by tools that will be built in the future."

I chose to keep the bit about tools that will be built in the future because there are currently no easily accessible tools that can surface this information, but they will be built post-upgrade either by the core team or maybe even community members.